Sources/Experiments/ExPlat.swift

```diff
@@ -105,9 +105,9 @@ import Cocoa
         case "control":
             return .control
         case "treatment":
-            return .treatment(nil)
-        default:
-            return .treatment(variation)
+            return .treatment
+        case let name:
+            return .customTreatment(name: name)
```

Comment:
Nitpick that might be out of scope for this PR, but this code seems like it would fit nicely in a custom initializer for `Variation`:

```
return Variation(from: variation)
```